subsys/spm/Kconfig

```diff
@@ -255,6 +255,67 @@ config SPM_NRF_DPPIC_NS
 	bool "DPPIC is Non-Secure"
 	default y
 
+# Start: nRF53 specific peripherals
```

Comment:
Instead of this, could we add an `if NRF5340` or whatever equivalent so that these are not visible for non-53 users.

Reply:
I agree, that's better! Some of the nRF9160 peripherals are not available for the nRF5340 from what I can understand, so `if nRF9150` could be added as well.